.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  release:
```

Comment:
We should aim for what we've set up or are in the process of setting up for our other repos:
- GHA workflow that enforces Conventional Commit tags in PR titles
- Push-button release workflow that automatically determines the next version number (and the release notes) based on Conventional Commit tags since previous version

Reply:
@ctreatma I added new PR with GHAs like in equinix-go #21